### PR TITLE
Centos 8 Fixes

### PR DESCRIPTION
**Pull request by zimmerry**

### CODE REVIEW

**Comment by zimmerry** [comment]
[CentOS 8: osl-imap](https://trello.com/c/TuUCU8GE/1890-centos-8-osl-imap)

File: test/integration/lmtp/inspec/lmtp_spec.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@
 end
 
 describe file '/var/log/maillog' do
-  its('content') { should match(/dovecot: lmtp\(foo\): .* saved mail to INBOX$/) }
+  its('content') { should match(/saved mail to INBOX$/) }
```
Comment (ramereth) [inline test/integration/lmtp/inspec/lmtp_spec.rb]: Why was this changed?

Comment (zimmerry) [reply test/integration/lmtp/inspec/lmtp_spec.rb]: That Inspec test fails on EL8 if it's unchanged. I think the logging may have changed between versions, but I'm not sure.

Comment (ramereth) [reply test/integration/lmtp/inspec/lmtp_spec.rb]: Does the log not show up in that file or does it show up somewhere else like in `journalctl`?

Comment (zimmerry) [reply test/integration/lmtp/inspec/lmtp_spec.rb]: There are some extraneous characters added into the log, and I'm not great at regex. Should be fixed with my latest commit.

**Review by ramereth — APPROVED** [review]
Please merge as-is.
